src/readme.md

```diff
@@ -0,0 +1,8 @@
+# Arcus for Azure Machine Learning
```

Comment:
I presume this README is to include it in the PyPi package?

Reply:
renamed to `package-description.md`just to be more clear